client/src/pages/Login/LoginForm/LoginForm.tsx

```diff
@@ -88,6 +89,7 @@ export default function Login({ handleSubmit }: Props): JSX.Element {
             <Button type="submit" size="large" variant="contained" color="primary" className={classes.submit}>
               {isSubmitting ? <CircularProgress style={{ color: 'white' }} /> : 'Login'}
             </Button>
+            <LogInWithDemoUser></LogInWithDemoUser>
```

Comment:
```suggestion
            <LogInWithDemoUser /> 
```

since theres no child component/content